Documenting --static flag for docs (#6962)
fishtown-analytics/docs.getdbt.com: website/docs/reference/commands/cmd-docs.md

dbt docs generate --empty-catalog
```

**Example**:

> Use the `--static` flag if you want to generate the docs as a static page for hosting on cloud storage provider. The contents of `catalog.json` and `manifest.json` will be inserted into the `index.html` file, creating a single page that can be easily shared via email or file sharing apps. 

Author: ```suggestion
Use the `--static` flag to generate the docs as a static page for hosting on a cloud storage provider. The `catalog.json` and `manifest.json` files will be inserted into the `index.html` file, creating a single page easily shared via email or file-sharing apps. 
```
